.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "OpenSCAD (Debian)",
+  "build": {
+    "dockerfile": "Dockerfile",
+    // Build docker container using the scripts directory as context,
+    // in order that it can copy the install dependencies script.
+    "context": "../scripts"
+  },
+
+  // Sanity check on startup.
+  // Intentionally executed outside of the docker build
+  // so that changes to required dependencies will be flagged
+  // and prompt users to rebuild the container.
+  "postStartCommand": "./scripts/check-dependencies.sh",
+
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        // C++ extension pack (includes CMake extension).
+        "ms-vscode.cpptools-extension-pack"
+      ]
+    }
+  },
+
+  // https://universal-blue.discourse.group/t/devcontainer-for-developing-an-application-with-a-gui/4634/4
+  // Configuration to allow OpenSCAD running inside the container to connect to host display.
+  "runArgs": [
+    "--device=/dev/dri",
+    "--env=WAYLAND_DISPLAY=${localEnv:WAYLAND_DISPLAY}",
+    "--network=host"
+    // "--env=XDG_RUNTIME_DIR=${localEnv:XDG_RUNTIME_DIR}",
+    // "--env=DBUS_SESSION_BUS_ADDRESS=unix:path=${localEnv:XDG_RUNTIME_DIR}/bus",
```

Comment:
I suggest either removing these or add comments explaining what they're for and when to enable them